src/main/scala/com/wavesplatform/transaction/protobuf/PBInternalImplicits.scala

```diff
@@ -0,0 +1,87 @@
+package com.wavesplatform.transaction.protobuf
+import com.google.protobuf.ByteString
+import com.wavesplatform.account.PublicKeyAccount
+import com.wavesplatform.account.protobuf.{Alias, Recipient}
+import com.wavesplatform.common.state.ByteStr
+
+private[protobuf] object PBInternalImplicits {
+  import com.google.protobuf.{ByteString => PBByteString}
+  import com.wavesplatform.account.{AddressOrAlias, Address => VAddress, Alias => VAlias}
+  import com.wavesplatform.common.utils._
+
+  implicit def byteStringToByteStr(bs: PBByteString): ByteStr = bs.toByteArray
+  implicit def byteStrToByteString(bs: ByteStr): PBByteString = PBByteString.copyFrom(bs)
+
+  implicit def fromAddressOrAlias(addressOrAlias: AddressOrAlias): Recipient = addressOrAlias match {
+    case a: VAddress => fromAddress(a)
+    case al: VAlias  => fromAlias(al)
+  }
+
+  implicit def fromAddress(address: VAddress): Recipient = {
+    Recipient.defaultInstance.withAddress(address.bytes)
+  }
+
+  implicit def fromAlias(alias: VAlias): Recipient = {
+    Recipient.defaultInstance.withAlias(Alias(alias.chainId: Byte, alias.name))
+  }
+
+  implicit class PBRecipientImplicitConversionOps(recipient: Recipient) {
+    def toAddress: VAddress = {
```

Comment:
`toAddress` should return `Either`, not throw an exception (so do other conversions)